universitat-mannheim-germanistische-linguistik.csl

```diff
@@ -0,0 +1,293 @@
+      </else>
+    </choose>
+  </macro>
+  <macro name="author-short">
+    <choose>
+      <if type="entry-dictionary" match="any">
+        <text variable="title"/>
+      </if>
+      <else>
+        <names variable="author" delimiter="/">
+          <name form="short" delimiter="/" initialize-with=". " delimiter-precedes-last="always"/>
+          <substitute>
+            <names variable="editor"/>
+            <names variable="translator"/>
+            <text macro="anon"/>
+          </substitute>
+        </names>
+      </else>
+    </choose>
+  </macro>
+  <macro name="access">
+    <choose>
+      <if variable="DOI">
+        <text variable="DOI" prefix="https://doi.org/"/>
+      </if>
+      <else-if match="any" variable="URL">
+        <group>
+          <text variable="URL"/>
+          <date variable="accessed" prefix="(" suffix=")">
+            <date-part name="day" form="numeric" suffix="."/>
+            <date-part name="month" form="numeric" suffix="."/>
+            <date-part name="year" form="long"/>
+          </date>
+        </group>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="title">
+    <text variable="title" text-case="title"/>
+    <choose>
+      <if type="book" variable="volume" match="all">
+        <text variable="volume" form="short" prefix=", Bd. "/>
+      </if>
+      <else-if type="book" variable="number-of-volumes" match="all">
+        <text variable="number-of-volumes" prefix=", " suffix=" Bände"/>
+      </else-if>
+    </choose>
+  </macro>
+  <macro name="genre">
+    <choose>
+      <if type="report" match="any">
+        <group suffix=".">
+          <text variable="genre" suffix=" "/>
+          <text variable="number"/>
+        </group>
+      </if>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <choose>
+      <if type="article-newspaper" match="none">
+        <group delimiter=": ">
+          <text variable="publisher-place"/>
+          <choose>
+            <if match="any" variable="publisher">
+              <text variable="publisher"/>
+            </if>
+            <else-if type="book" match="any">
+              <text value="o. V."/>
+            </else-if>
+          </choose>
+        </group>
+      </if>
+    </choose>
+    <choose>
+      <if type="entry-dictionary" match="any">
+        <text macro="year-date" prefix=", "/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="year-date">
+    <group delimiter="/">
+      <date date-parts="year" form="text" variable="original-date"/>
+      <choose>
+        <if variable="issued">
+          <date variable="issued">
+            <date-part name="year"/>
+          </date>
+        </if>
+        <else>
+          <text term="no date" form="short"/>
+        </else>
+      </choose>
+    </group>
+  </macro>
+  <macro name="locator">
+    <choose>
+      <if type="article-journal">
+        <choose>
+          <if variable="volume" match="none">
+            <text variable="DOI" prefix="DOI: "/>
```

Comment:
This seems odd? [Above](https://github.com/citation-style-language/styles/pull/5228/files#diff-32f81e0fcfd685671f7aaca703b7ea32a6bc49f691674810465684aec1d12094R78) you use the "correctly" displayed DOI, here with the much less useful DOI: prefix -- is that on purpose? If so, could you convince them to change this in the guide?

Reply:
Oh yes, that is more like a left-over from the template style used here. For journal articles the volume should always been shown and if the article is an online article, then additionally the URL. That is what the documentation says. I extended this to use the DOI instead of the URL whenever possible. -- This is fixed in the new version.